Dockerfile

```
@@ -38,58 +63,25 @@ RUN apk --no-cache --quiet add $DEPS
 RUN pip install --no-cache-dir --upgrade pip setuptools
 
 ###
-# Install dependencies
+# Install Python dependencies
 ###
 RUN pip install --no-cache-dir --upgrade \
     https://github.com/cisagov/mongo-db-from-config/tarball/develop \
     pytz
 
-
-###
-# Setup the saver user and its home directory
-###
-FROM install AS setup_user
-
 ###
-# Dependencies
-#
-# Install shadow, so we have adduser and addgroup.
-#
-# Setup user dependencies are only needed for setting up the user and
-# will be removed at the end of that process.
-###
-ENV SETUP_USER_DEPS \
-    shadow
-RUN apk --no-cache --quiet add $SETUP_USER_DEPS
-
-###
-# Create unprivileged user
-###
-RUN addgroup -S $USER
-RUN adduser -S -g "$USER user" -G $USER $USER
-
-###
-# Remove build dependencies
-###
-RUN apk --no-cache --quiet del $SETUP_USER_DEPS
-
 # Put this just before we change users because the copy (and every
 # step after it) will always be rerun by docker, but we need to be
 # root for the chown command.
-COPY . $HOME
-RUN chown -R ${USER}:${USER} $HOME
-
-
-###
-# Setup working directory and entrypoint
 ###
-FROM setup_user AS final
+COPY src ${CISA_HOME}
```

So we only bring in the files we actually need.

```suggestion
COPY src/*.py src/*.sh ${CISA_HOME}
COPY src/include ${CISA_HOME}/include
```